pkg/windows/portable.py

```diff
@@ -29,16 +30,12 @@ def display_help():
 
 def main(argv):
     target = ''
-    python_dir = 'Python{0}{1}'.format(sys.version_info[0], sys.version_info[1])
-    if sys.version_info >= (3, 5):
-        from win32com.shell import shellcon, shell
-        search = shell.SHGetFolderPath(0, shellcon.CSIDL_PROGRAM_FILES, 0, 0)
-        search = os.path.join(search, python_dir)
-    else:
-        search = os.path.join('C:\\', python_dir)
+    search = sys.exec_prefix
```

Comment:
`sys.exec_prefix` happens to work on Windows right now, but its meaning isn't exactly what we want:
https://docs.python.org/3.6/library/sys.html#sys.exec_prefix
eg, it translates to `/usr/local` on some Linux distributions which is definitely not the path to the Python executable.

I would prefer a more direct way of getting the path to the Python executable, for instance:

```
search = os.path.dirname(sys.executable)
```

Reply:
Done